tests/test_ParallelRestart.cpp

```diff
@@ -571,6 +571,17 @@ BOOST_AUTO_TEST_CASE(Tabdims)
 }
 
 
+BOOST_AUTO_TEST_CASE(EndpointScaling)
+{
+#if HAVE_MPI
+    Opm::EndpointScaling val1(std::bitset<4>(123456));
```

Comment:
This is an unexpected initial value for a bitset of size 4.  From reference material I know that it's perfectly well defined (and creates an all-ones bitset), but I still had to think twice about it.

Reply:
right, i can make it a valid value.

Reply:
> right, i can make it a valid value.

Cool.  You might even consider constructing the bitset from a string:
```C++
std::bitset<4>("1111")
```